vuepress/docs/docs/client.md

```diff
@@ -32,16 +32,53 @@ addSbtPlugin("com.github.ghostdogpr" % "caliban-codegen-sbt" % "1.0.1")
 
 And enable it in your `build.sbt` file:
 ```scala
-enablePlugins(CodegenPlugin)
+enablePlugins(CalibanPlugin)
 ```
 
-Then call the `calibanGenClient` sbt command.
+At this point, the `caliban` command will cause any files in `src/main/graphql` to be translated into a Caliban-generated client library. This happens automatically any time you `compile`.
```

Comment:
Is this configurable? What if you want to use an API URL?

Reply:
Not possible, with how the plugin is currently set up. I think for that, adding `calibanSetting(url: URL)(...)` would work.

Reply:
OK, I've made this change. To your original question, the path is configurable via `calibanSources`